src/api/apollo.js

```diff
@@ -72,9 +153,7 @@ export default function createApolloClient({
 		assumeImmutableResults: true,
 	});
 
-	// set default local state
-	cache.writeData({ data: defaults });
-	client.onResetStore(() => cache.writeData({ data: defaults }));
```

Comment:
This default local state still needs to be set. Since we can't use `writeData` anymore, this will be a bit more complicated to do. I would recommend:

- changing `defaults` in each localResolver file to be a function that accepts the `cache` as an argument and calls `cache.writeQuery` with the data
- creating a function that calls `defaults(cache)` for every localResolver file
- calling that function here and in the client.onResetStore callback